python/lsst/shapelet/shapeletFunction/shapeletFunctionContinued.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import, division, print_function
+
+from .shapeletFunction import ShapeletFunction
+
+from lsst.utils import continueClass
+
+__all__ = []
+
+@continueClass
+class ShapeletFunction:
+    def __reduce__(self):
+        return (ShapeletFunction, (self.getOrder(), self.getBasisType(),
+            self.getEllipse(), self.getCoefficients()))
```

Comment:
This and all other new `xContinued.py` functions contain trivial flake8 errors; please clean them up:
- a flake8 complaint about redefining the class; I assume a `# noqa F811` will be required to quell this warning.
- add an extra blank line before `@continueclass`
- remove the extra blank line at the end (a final carriage return is wanted, but not two of them)
`autopep8` will clean up the last two items.